components/sharing/SocialShareModal.vue

```diff
@@ -36,6 +68,41 @@
       mediaUrl: {
         type: String,
         default: null
+      },
+      identifier: {
+        type: String,
+        default: null
+      }
+    },
+
+    data() {
+      return {
+        oEmbedDataHtml: null,
+        embedCopied: false,
+        onItemPage: !!this.identifier
+      };
+    },
+
+    methods: {
+      requestEmbed() {
+        oEmbedForEndpoint(process.env.OEMBED_ENDPOINT || 'https://oembedjs.europeana.eu/',
+                          `${EUROPEANA_DATA_URL}/item${this.identifier}`)
+          .then((response) => {
+            if (response.data.html) {
+              this.oEmbedDataHtml = response.data.html;
+            }
+          });
+      },
+      copyEmbedCode(event) {
+        if (event.type === 'click' || event.keyCode === 13) {
+          let textarea = this.$refs.shareEmbed;
```

Comment:
This doesn't warrant a variable assignment. Just do `this.$refs.shareEmbed.select();`